src/main/java/Duke.java

```diff
@@ -1,10 +1,116 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
+    public  Duke(){
+
+    }
+    ArrayList<Task> tasks = new ArrayList<>();
+    private final static String horLine = "------------------------------------------------------";
+    private final static String welcomeMsg = horLine + "\nHello! I'm Duke\n" +
+            "What can I do for you?\n" + horLine;
+
+    private void addToList(String taskName) {
+        try{
+            //if task is a todo
+            if (taskName.matches("\\btodo\\s.*\\b")){
+                Task newTask = new ToDo(taskName.substring(5));
```

Comment:
I feel that instead of `newTask` you can name it something else to indicate that it's a `ToDo` class instead of a normal `Task` object (perhaps newToDo). However, this is not part of the coding standards, so feel free to close this